v2/tools/generator/internal/codegen/pipeline/json_serialization_test_cases.go

```diff
@@ -45,7 +45,7 @@ func InjectJsonSerializationTests(idFactory astmodel.IdentifierFactory) Stage {
 			return state.WithTypes(state.Types().OverlayWith(modifiedTypes)), nil
 		})
 
-	return stage.RequiresPostrequisiteStages("simplifyDefinitions" /* needs flags */)
+	return stage.RequiresPostrequisiteStages("simplifyDefinitions", CreateResourceExtensionsStageID /* needs flags */)
```

Comment:
The reason we have both prerequisite and postrequisite stages is so that we can put the restriction on the stage that cares. In this case, it's the `CreateResourceExtensionsStage` that cares about not having tests; the test stages don't give two hoots.